services/s3/src/it/java/software/amazon/awssdk/services/s3/UserMetadataIntegrationTest.java

```diff
@@ -83,5 +93,18 @@ public void putObject_PutsUserMetadata() throws Exception {
         Map<String, String> returnedMetadata = response.metadata();
 
         assertThat(returnedMetadata).containsAllEntriesOf(userMetadata);
+        assertThat(returnedMetadata).containsKey(metadataKey);
+    }
+
+    @Test
+    public void userMetadata_caseInsensitive() {
+        String caseInsensitiveMetadata = "";
+        HeadObjectResponse response =
+            s3.headObject(b -> b.bucket(BUCKET_NAME).key(KEY).overrideConfiguration(r -> r.putHeader("x-amz-metadata-test",
+                                                                                            "test")));
+
+        //assertThat(response.metadata())
```

Comment:
Remove?

Reply:
Yes, this test is actually unnecessary and I forgot to remove it.